view/base/web/js/model/address_validation_core.js

```diff
@@ -105,6 +105,17 @@ function (ko, $) {
             this.suggestedAddresses(addr);
         },
 
+        isValidAddress: function (address) {
+            return !!(
+                address &&
+                address.country_id === 'US' &&
```

Comment:
This method needs to only validate that the country_id is set, not that it is "US". In it's current state it prevents the logic that validates the country_id is US from ever being reached.

Reply:
You are correct. I updated.